json/test_asdf.d

```diff
@@ -0,0 +1,54 @@
+/+dub.json:
+{
+    "name": "json_d_asdf",
+    "dependencies": {
+        "asdf": {"path" : "../../asdf"}
```

Comment:
Should be package name and version from the dub repository instead

Reply:
Ah, yes. OK, i will fix it after the next asdf beta. plz wait with this PR